server/src/routes/users.ts

```diff
@@ -413,6 +414,19 @@ router.get(
 	})
 );
 
+router.get(
+	'/match_suggestions',
+	sessionExtractor,
+	asyncHandler(async (req: CustomRequest, res) => {
+		if (!req.session || !req.session.userId) throw new AppError(`Please log in first`, 400);
+		if (!(await getAndUpdateUserCompletnessById(req.session.userId))) throw new AppError('Please, complete your own profile first', 400);
```

Comment:
Maybe better to place it inside service getInitialMatchSuggestionsIds(). But maybe not XD

Reply:
Right, sure
I missed that point
If it is usually called in services, then let's move it to service, yes
Consistency is the key